### ISSUE TITLE
[BUG] aztfmod/azurecaf Key Expired

### ISSUE THREAD

**Issue by Paxilein** [claranet/terraform-azurerm-vnet-peering#6]
### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

The PGP Key for aztfmod/azurecaf expired and is not maintained anymore... Time to change claranet to something else!?

### Terraform Version

1.6.0

### AzureRM Provider Version

3.115.0

### Affected Resource(s)/Data Source(s)

claranet/vnet-peering/azurerm

### Terraform Configuration Files

```hcl
N/A
```

### Debug Output/Panic Output

```shell
│ Error: Failed to install provider
│ 
│ Error while installing aztfmod/azurecaf v1.2.28: error checking signature:
│ openpgp: key expired
```

### Expected Behaviour

Create peering

### Actual Behaviour

Fail on init

### References

https://github.com/claranet/terraform-azurerm-vnet-peering/blob/master/versions.tf

https://github.com/aztfmod/terraform-provider-azurecaf/issues/286

**Comment by Shr3ps** [comment]
Thanks for your issue,

It seems yes it expires on the Hashicorp Terraform registry, but at Claranet we moved to OpenTofu and its registry (no issue so far).
And we don't know if we will still maintain Terraform compatibility

**Comment by Paxilein** [comment]
So, what's the suggested way to approach this? I am using the Claranet Vnet Peering in my TF code and changing everything to OpenTofu just for this thing is not going to happen... 😅

**Comment by Shr3ps** [comment]
> So, what's the suggested way to approach this? I am using the Claranet Vnet Peering in my TF code and changing everything to OpenTofu just for this thing is not going to happen... 😅

We're checking if we can fork the provider.

On the other hand what do you mean by "changing everything to OpenTofu just for this thing is not going to happen..." ?
Using OpenTofu is just a binary change, retrocompatible with your code. You only have to download/install it,
trigger `tofu init` and `tofu plan/apply`, and everything will works smoothly (tofu 1.6 is compatible with HCL/Terraform 1.5 and 1.6).

**Comment by Shr3ps** [comment]
This issue is now fixed with latest v7.1.0  - https://github.com/claranet/terraform-azurerm-vnet-peering/blob/master/CHANGELOG.md#710-2024-10-03
